packages/bap-ida/bap-ida.1.0.0~alpha/opam

```diff
@@ -24,6 +24,7 @@ remove: [
 ]
 depends: [
          "conf-ida"
+         "bap-ida-python"
```

Comment:
This was there earlier itself. It is because `lib/bap_ida/bap_ida.ml` has an `open Core_kernel.Std` etc.

The only change I have made in this file is to make `bap-ida` depend on `bap-ida-python`.